sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/SessionRecords/CertificateClientLiveTests/DownloadECDsaCertificateSignLocalVerifyRemote(application%x-pem-file,P-256).json

```diff
@@ -5,7 +5,7 @@
       "RequestMethod": "POST",
       "RequestHeaders": {
         "Accept": "application/json",
-        "Content-Type": "application/json",
```

Comment:
I'm concerned all these are missing. If the service returns it, the proxy should not modify the content (sans Content-Length, depending on why - separate comment).

Reply:
this is in the Request headers.

Reply:
Good point, though I wonder how they got there in the first place. I certainly didn't add them. Did the old recording pipeline (or even HTTP pipeline) add it by default?

/cc @pakrym

Reply:
Is the generated client adding them?

Reply:
We don't use generators for KV except for the Administration package. Was nowhere near ready at the time. But that's also how I know this didn't change. It's all hand-coded, and I'm the only one who's touched the pipeline (a thin abstraction over Core's) in ages.

Reply:
Well, the transports (HttpClient and HttpWebRequest) removes the content-type from these requests. We just didn't know it before because our test recordings were saved prior to the transport.